src/app/core/auth/components/login/login.component.html

```diff
@@ -35,11 +35,20 @@ <h2 class="title">Welcome</h2>
               <mat-checkbox color="primary" class="font14"
                 >Remeber me</mat-checkbox
               >
+              <a href="#" class="forgot-password" *ngIf="!isAdmin"
+                >Forgot Password?</a
+              >
             </div>
           </div>
           <div class="col-12 animation-target">
             <button mat-flat-button color="primary" class="w-100">Login</button>
           </div>
+          <div class="col-12 animation-target mt-5" *ngIf="!isAdmin">
+            <p class="account">
+              Don't have an account?
+              <a href="#" class="register">Register Now!</a>
```

Comment:
there is no logic for register page redirection

Reply:
That was done by rajvi ma'am